src/pages/user.tsx

```diff
@@ -124,6 +129,12 @@ export function User() {
           <UserCard data={user} key={user.id} />
         ))}
       </div>
+      {manageUserModalVisibility && (
```

Comment:
Could you please replace this short circuit conditional to a full ternary?

Reply:
i just did that, thank you for pointing